src/main/resources/application-q1.properties

```diff
@@ -3,7 +3,7 @@
 fss-gateway-app-id=${FSS_GATEWAY_APP_ID:dev-fss:pensjonselvbetjening:pensjon-selvbetjening-fss-gateway-q1}
 fullmakt.url=${FULLMAKT_URL:https://pensjon-selvbetjening-fss-gateway-q1.dev.intern.nav.no}
 pdl.url=${PDL_URL:https://pensjon-selvbetjening-fss-gateway-q1.dev.intern.nav.no}
-pen.url=${PEN_URL:https://pensjon-selvbetjening-fss-gateway-q1.dev.intern.nav.no}
+pen.url=${PEN_URL:https://pensjon-pen-q1.dev-fss-pub.nais.io}
```

Comment:
Er dette et miljø vi kjører i gcp? 😱

Reply:
Nei, kan egentlig ryddes opp i. Kun Q2 for opptjening i dev.